SPDX-License-Identifier: LGPL-2.1-or-later

Preserve compiler flags supplied by the ports framework.  Avoid adding
the platform definitions twice; common.mf already appends them.

Index: mf/Makefile.OpenBSD
--- mf/Makefile.OpenBSD.orig
+++ mf/Makefile.OpenBSD
@@ -8,10 +8,10 @@ APPLRESDIR = /usr/X11R6/lib/X11/app-defaults
 INCDIRS = -I/usr/X11R6/include -I/usr/local/include
 LIBDIRS = -L/usr/X11R6/lib -L/usr/local/lib
 
-CFLAGS = -O2 -Wall $(INCDIRS)
+CFLAGS += -Wall
 
 .c.o:
-	$(CC) $(CFLAGS) $(DEFINES) $(INCDIRS) -c -o $(@D)/$(@F) $<
+	$(CC) $(CFLAGS) $(INCDIRS) -c -o $(@D)/$(@F) $<
 
 include common.mf
 
